flink-formats/flink-avro/src/test/java/org/apache/flink/formats/avro/AvroKryoSerializerRegistrationsTest.java

```diff
@@ -56,8 +56,9 @@
 	public void testDefaultKryoRegisteredClassesDidNotChange() throws Exception {
 		final Kryo kryo = new KryoSerializer<>(Integer.class, new ExecutionConfig()).getKryo();
 
-		try (BufferedReader reader = new BufferedReader(new InputStreamReader(
```

Comment:
I think changes like this one are not necessary. The `BufferedReader` forwards the `close()` call to the nested `InputStreamReader`. So there is no need to have them separate in the `try(...)` clause.

I would keep the original pattern, which seems easier to read/format.